yb-voyager/src/pgss/parser.go

```diff
@@ -58,12 +58,13 @@ func ParseFromCSV(csvPath string) ([]PgStatStatements, error) {
 		}
 
 		// Entry is valid if parsing succeeded
-		entries = append(entries, *entry)
+		entries = append(entries, entry)
 		lineNumber++
 	}
 
 	log.Infof("PGSS CSV parsing completed with %d entries", len(entries))
-	return entries, nil
+	// since there is a possibility of same queryid for different entries(due to userid difference), we need to merge them
```

Comment:
the merge is based on query, not queryid, right? Pls fix the comment. 
Also, nit: maybe let's call the function `MergePgStatStatementsBasedOnQuery` ?